specs/query/oslc-query.html

```diff
@@ -409,39 +432,39 @@ <h2>Discovering a Query Capability</h2>
   <section>
     <h2>Using a Query Capability</h2>
 
-    Executing a query or search is achieved by executing either of the following:
+    Executing a query or search is achieved by performing either of the following:
     <ul>
       <li>a <code>GET</code> on the <i>query base</i> URI of a query capability, with zero, one or
-        more query parameters shown in the table below.
-        <p class="conformance">Servers MUST support a GET on the <i>query base</i> URI of a query capability.</p>
-        This is the simplest usage, but clients must be careful to avoid constructing URIs
-        whose length exceeds server limits.
-        <p class="conformance">If such a GET URI exceeds 1000 characters, clients SHOULD use the POST mechanism
-          described below.</p>
+        more query parameters shown in <a href="#Table1">Table 1</a>.
       </li>
       <li>a <code>POST</code> on the <i>query base</i> URI of a query capability with a
         <code>application/x-www-form-urlencoded</code> body containing zero, one or
-        more query parameters shown in the table below.
-        <p class="conformance">Servers SHOULD support <code>POST</code> on the <i>query base</i> URI of a query
-          capability
-          to avoid clients having to construct URIs that are too long.</p>
+        more query parameters shown in <a href="#Table1">Table 1</a>.
+
       </li>
     </ul>
-
+   
+    <p class="conformance">Servers MUST support a GET on the <i>query base</i> URI of a query capability.</p>
+        
+    <p class="conformance">Servers SHOULD support POST on the query base URI of a query capability to avoid
+      clients having to construct URIs that are too long for a GET.</p>
+      
+    <p class="conformance">A <code>GET</code> or <code>POST</code> request on a query capability MAY use zero, 
+      one or more of the OSLC query parameters listed in <a href="#Table1">Table 1</a>.</p> 
+
+    <p class="conformance">Clients SHOULD use the POST mechanism described above if the URI of a GET
```

Comment:
should not be a conformance clause, we don't specify requirements for clients, only servers

Reply:
Fixed in today's commit.